rosie/federal_senate/adapter.py

```diff
@@ -0,0 +1,50 @@
+import os
+
```

Comment:
💅 If we add a line break to separate native (`os`) modules from thrid-party ones (`panda` and `numpy`) I'd also add a line to separate third-party from our package's modules (`serenata_toolbox.…`).